fix command when not passing arguments to `capture`

The program panics when running `harpoon capture -f main.doSomething` without specifying a command to run.
Eg.
```
panic: runtime error: index out of range [0] with length 0                                                                                                                                                                                    
                                                                                                                                                                                                                                              
goroutine 1 [running]:                                                                                                                                                                                                                        
github.com/alegrey91/harpoon/cmd.glob..func1(0xc000148100?, {0xc00007a140, 0x0, 0x5})                                                                                                                                                         
        /home/runner/work/harpoon/harpoon/cmd/capture.go:113 +0x13f5                                                                                                                                                                          
github.com/spf13/cobra.(*Command).execute(0x874020, {0xc00007a0f0, 0x5, 0x5})                                                                                                                                                                 
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:987 +0xaa3                                                                                                                                                           
github.com/spf13/cobra.(*Command).ExecuteC(0x874300)                                                                                                                                                                                          
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff                                                                                                                                                          
github.com/spf13/cobra.(*Command).Execute(...)                                                                                                                                                                                                
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039                                                                                                                                                                 
github.com/alegrey91/harpoon/cmd.Execute()                                                                                                                                                                                                    
        /home/runner/work/harpoon/harpoon/cmd/root.go:37 +0x1a                                                                                                                                                                                
main.main()                                                                                                            
        /home/runner/work/harpoon/harpoon/main.go:21 +0xf
```

This is because the code is not checking if we have enough arguments in input to run.

--- a/internal/ebpf/probesfacade/captor/capture.go
+++ b/internal/ebpf/probesfacade/captor/capture.go
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"context"
 	"encoding/binary"
+	"errors"
 	"fmt"
 	"path/filepath"
 	"sync"
@@ -51,6 +52,10 @@ type ebpfSetup struct {
 // to the ebpf program.
 // Returns the ebpfSetup struct in case of seccess, an error in case of failure.
 func InitProbes(functionSymbol string, cmdArgs []string, opts CaptureOptions) (*ebpfSetup, error) {
+	if len(cmdArgs) == 0 {
+		return nil, errors.New("error no arguments provided, at least 1 argument is required")
+	}
+
 	if !opts.LibbpfOutput {
 		// suppress libbpf log ouput
 		bpf.SetLoggerCbs(

--- a/tests/integration.txtar
+++ b/tests/integration.txtar
@@ -35,6 +35,10 @@ exec harpoon capture -e -c -f main.main -- ./bin/example-app
 stderr 'command execution error: fork/exec .* permission denied'
 exec chmod +x bin/example-app
 
+# test it doesn't panic when no arguments are provided
+! exec harpoon capture -e -c -f main.main
+stdout '.* error no arguments provided, at least 1 argument is required'
+
 # setting up test application
 exists bin/example-app
 # this must be done within the testscript environment

